vkconfig/dlgprofileeditor.cpp

```diff
@@ -154,7 +154,7 @@ dlgProfileEditor::dlgProfileEditor(QWidget *parent, Configuration *configuration
     }
 
     ui_->lineEditName->setText(configuration_->name);
-    ui_->lineEditDesc->setText(configuration_->description);
+    //ui_->lineEditDesc->setText(configuration_->description);
```

Comment:
Let's not check in commented-out source if at all possible.

Reply:
Fixed